src/components/slides/slide-list/SlideList.tsx

```diff
@@ -0,0 +1,15 @@
+import { slides } from '@/data/Slides';
+import React from 'react';
+import SlideItem from '../slide-item/SlideItem';
+
+const SlideList = () => {
+	return (
+		<>
+			{slides.map((slide) => (
+				<SlideItem key={Math.random() * 20 + slide.subtitle} {...slide} />
```

Comment:
because it is not recommended and it's bad practice which can lead some issue in future
https://www.google.com/search?q=react+is+using+array+index+to+define+key+is+good&oq=reac&aqs=chrome.1.69i60j69i59l3j69i61l2j69i65j69i60.1787j0j7&sourceid=chrome&ie=UTF-8

i think the best way is use crypto.randomUUID() or uuid library

Reply:
> **nitpick:** Why not using just index from your map function?

because it is not recommended and it's bad practice which can lead some issue in future
https://www.google.com/search?q=react+is+using+array+index+to+define+key+is+good&oq=reac&aqs=chrome.1.69i60j69i59l3j69i61l2j69i65j69i60.1787j0j7&sourceid=chrome&ie=UTF-8

i think the best way is use crypto.randomUUID() or uuid library